src/Analyzers/CSharp/Tests/RemoveUnnecessaryCast/RemoveUnnecessaryCastTests.cs

```diff
@@ -12769,6 +12769,31 @@ static Delegate M(object o)
             }.RunAsync();
         }
 
+
+        [WorkItem(60248, "https://github.com/dotnet/roslyn/issues/60248")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnnecessaryCast)]
+        public async Task RemoveCastInTopLevelPrograms()
+        {
+            var test = new VerifyCS.Test()
+            {
+                TestCode = @"
+int x = 1;
+int y = [|(int)|]x;
+",
+                FixedCode = @"
+int x = 1;
+int y = x;
+",
+                LanguageVersion = LanguageVersion.CSharp10,
+                TestState =
+                {
+                    OutputKind = OutputKind.ConsoleApplication,
+                },
+            };
+
+            await test.RunAsync();
+          }
```

Comment:
Is indentation bad here? I'm looking from mobile so not sure

Reply:
Yes, the indentation is bad.